app/Livewire/Server/ValidateAndInstall.php

```
@@ -206,6 +206,9 @@ public function validateDockerVersion()
                 if (! $proxyShouldRun) {
                     return;
                 }
+                // Ensure networks exist BEFORE dispatching async proxy startup
+                // This prevents race condition where proxy tries to start before networks are created
+                instant_remote_process(ensureProxyNetworksExist($this->server)->toArray(), $this->server, false);
                 StartProxy::dispatch($this->server);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Actions\Proxy\StartProxy' in method 'validateDockerVersion'. (undefined)

(StaticAccess)